html/includes/home-header.html

```diff
@@ -13,10 +13,10 @@ <h4 class="white font-weight bold pb-4">Developer Tooling, Cloud, and Market</h4
           <a href="develop.html" class="btn mr-1 ml-1 btn-primary btn-lg mb-2 btn-round btn-feature" role="button" aria-pressed="true">Build Amazing Apps</a>
          </div>
          <div class="col-12 col-lg-4 bitbox-hero text-center">
-          <h3 class="academy-title pt-5"><i class="fa fa-cube" aria-hidden="true"></i> BITBOX SDK</h3>
-          <p class="academy-p white">36,000+ downloads</p>
-          <p class="academy-p white">100+ countries</p>
-          <p class='white'>INSTALL VIA NPM</p>
+          <h3 class="academy-title pt-5"><a href="http://bitbox.earth/" target="_blank"><i class="fa fa-cube" aria-hidden="true"></i> BITBOX SDK</a></h3>
```

Comment:
Can you link to https://developer.bitcoin.com/bitbox.html? It's more up to date than bitbox.earth.

Reply:
Revised and fixed merge conflicts